code/modules/mob/living/simple_animal/bot/cleanbot.dm

```diff
@@ -130,7 +130,8 @@
 		target = scan(/obj/effect/decal/remains)
 
 	if(!target && trash) //Then for trash.
-		target = scan(/obj/item/trash)
+		target = scan(/obj/item/trash, /obj/item/cigbutt, /obj/item/ammo_casing/spent, /obj/item/light/tube/broken, 
```

Comment:
this doesn't do what you think it does.

Reply:
https://github.com/tgstation/tgstation/blob/a9ca93b60dbf7b9f0fc5cbe94c406a2126312fee/code/modules/mob/living/simple_animal/bot/bot.dm#L403

Reply:
change to /obj/item I'd think